src/main/java/space/obminyashka/items_exchange/rest/api/ApiKey.java

```diff
@@ -42,6 +42,9 @@ public class ApiKey {
     public static final String LOCATION_DISTRICT = LOCATION + "/district";
     public static final String LOCATION_CITY = LOCATION + "/city";
     public static final String LOCATIONS_INIT = LOCATION + "/locations-init";
+    public static final String LOCATIONS_INIT_AREA = LOCATION + "/area";
+    public static final String LOCATIONS_INIT_DISTRICT = LOCATION + "/district";
+    public static final String LOCATIONS_INIT_CITY = LOCATION + "/city";
```

Comment:
I see we already have LOCATION + "area" (line 41), "/district" (line 42),  and "city" (line 43) and it looks like your new constants could be removed because they are absolutely identical to those above.